lib/tests/suites/argonaut_sprint_1_test.rb

```diff
@@ -65,13 +70,13 @@ def define_metadata(method)
           validates resource: "Patient", methods: ["read", "search"]
         }
 
-        @id = @client.read_feed(@rc).resource.entry.first.resource.xmlId
```

Comment:
rather than removing this perhaps we should rescue to patient_id = nil then apply the next line?